### ISSUE TITLE
[Adaptive Cards - Expense Report.json]: VoiceOver is not announcing table and column header information for the Data table present under 'ExpenseReport.json' card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#48]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on  "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "Expense Report.json" control and select it.
7. Navigate the 'Expense Report.json' screen and enable Voiceover for the ease of access.
8. Navigate to the rows present under data table and hear screen reader narration.

**Actual Result:**
VoiceOver is not announcing table and column header information for the Data table present under 'ExpenseReport.json' card. Voiceover is announcing as, "06-19 text field double tap to edit Use the rotor to access Misspelt Words.".

**Expected Result:**
Voiceover focus should announce table and column header information for the Data table. Voiceover should only announce as, "Column header Date 06-19 Use the rotor to access the data".

**User Experience:**
It will impact the screen reader user if voiceover is not announcing table and column header information for the Data table present under 'ExpenseReport.json' card.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/e3987220-697c-4d39-a3ee-47d6673236b6
